app/styles/addon.css

```diff
@@ -1,8 +1,8 @@
 .json-pretty pre {
-    outline: 1px solid #bbb;
```

Comment:
nitpick: whitespace changes seem unrelated; ideally we just copy the original file so it doesn't appear in the diff as a change

Reply:
That is strange, I moved the file, didnt touch the contents

Reply:
Ahhh, my environment is set up to use tabs vs. spaces

Reply:
Would like like me to try to revert?

Reply:
If you wouldn't mind, yes please! As an aside, we could add the `.editorconfig` from the addon blueprints: https://github.com/ember-cli/ember-addon-output/blob/master/.editorconfig

Reply:
Should be all set